### ISSUE TITLE
Create a spreadsheet template for accessibility audit results

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/dacs_handbook#68]
Some examples:

- Section 6 of [the WCAG-EM template](https://www.w3.org/WAI/test-evaluate/report-template/)
- @christinach's audit of the catalog: [spreadsheet](https://docs.google.com/spreadsheets/d/1lV8AyTh1VZUrx-41s2impPCIn722MjRHJ_bvgLScuLs/edit#gid=825091091) / [full report](https://docs.google.com/document/d/1EFV8OB5gZ8l3YXSs_badUVX8DT-SNLS_vcEF0mpZkeY/edit#heading=h.98x71ddg0kjn)
- [Key sites audit results](https://docs.google.com/spreadsheets/d/14glZXKkH0hb3RBXrZhgXtrUI1DpkulH4eoorhzrUY6s/edit#gid=1992549741)

**Comment by sandbergja** [comment]
Here's [a spreadsheet based on @christinach's orangelight audit spreadsheet and OIT's key sites spreadsheet](https://docs.google.com/spreadsheets/d/1xTNkQI-rHp0LzyjtBDRGtHFNdyG454BihGmjl71N2bA/edit#gid=0).  We used it in https://github.com/pulibrary/orangelight/issues/3564, and it seemed to work okay.  @christinach , what do you think?

**Comment by christinach** [comment]
@sandbergja it look great! Can we add a column to include github tickets.
@ishasinha1 pointed out that you cannot add more than one options in the success criterion column.
@lizgarciao also pointed out that we are missing the github tickets column.

**Comment by christinach** [comment]
We added the extra column and updated the column 'issue' title.
